### PR TITLE
[WIP] test cleanup

### PR DESCRIPTION
**Pull request by dsyme**
This is a cleanup of the "fsharp" suite, to make it simpler

Still to be finalized, submitting for more complete CI

### CODE REVIEW

File: src/FSharpSource.targets [hunk 1]
```diff
@@ -111,6 +111,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(TargetFramework)'=='net40'">
+    <DefineConstants>$(DefineConstants);FX_ATLEAST_45</DefineConstants>
```
Comment (KevinRansom) [inline src/FSharpSource.targets]: Probably not at least_45, since it is supposed to run on dotnet40

Comment (KevinRansom) [inline src/FSharpSource.targets]: win 7 is 40.

**Comment by mmitche** [comment]
@dotnet-bot test this please (changed hooks)

**Comment by dsyme** [comment]
@KevinRansom A heads up that the changes delete the batch files for the "fsharp" suite (hence the 9000 line removal)

I'm working out how to compile the "fsharp" suite driver code for .NET Core, When run in that mode, can invoke the .NET Core F# compiler to run most of the tests (same coverage as the FSC_CORE permutation).  

I'm trying to reduce the use of permutations and make things a bit more explicit.

Also am trying to allow those tests to be run from the F# REPL in scripting mode, something which has worked well for F# Compiler Service testing.

**Comment by dsyme** [comment]
Also addresses https://github.com/Microsoft/visualfsharp/issues/889

**Pull request merged**